embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/arch/schema/EmbType.kt

```diff
@@ -35,7 +35,12 @@ internal sealed class EmbType(type: String, subtype: String?) : TelemetryType {
 
         internal object Breadcrumb : System("breadcrumb")
 
-        internal object Log : System("log")
+        internal object Log : System("log") {
+            /**
+             * Attribute name for a unique id identifying the log
+             */
+            val embLogId = EmbraceAttributeKey("log_id")
```

Comment:
Is this necessary? What would it look like in production & has this been discussed with backend folks? My understanding is that we can use the inbuilt IDs generated by OTel and that our own ID/unique name isn't required for logs as long as we add an `emb.type`

Reply:
This is just what was being written previously. @lucaslabari?

 If it's not necessary and can be removed, I can do that now or separately

Reply:
Let me validate with backend and see if it's necessary.